packages/twenty-front/src/modules/settings/data-model/constants/SettingsCompositeFieldTypeConfigs.ts

```diff
@@ -28,8 +28,9 @@ import { FieldMetadataType } from '~/generated-metadata/graphql';
 export type SettingsCompositeFieldTypeConfig<T> = SettingsFieldTypeConfig<T> & {
   subFields: (keyof T)[];
   filterableSubFields: (keyof T)[];
+  importableSubFields?: (keyof T)[];
```

Comment:
why not simplify the API by providing a CompositeSubFieldConfig type
{ 
subFieldName  
 subFieldLabel:
   isFilterable
   isImportable
}

Reply:
and why not T => CompositeFieldType